src/loader/multi_loader.rs

```diff
@@ -0,0 +1,70 @@
+use crate::Loader;
+use fluent_bundle::FluentValue;
+use std::borrow::Cow;
+use std::collections::HashMap;
+
+pub use unic_langid::LanguageIdentifier;
+
+/// A loader comprised of other loaders.
+///
+/// This loader allows for loaders with multiple sources to be used
+/// from a single one, instead of a multiple of them.
+pub struct MultiLoader {
+    pub loaders: Vec<Box<dyn Loader>>,
```

Comment:
I would make this field private.

Also is `Vec` the right data structure here? I think it would be better to use `VecDeque` to allow people to add new loaders with the highest preference, and add loaders with the lowest preference.

Reply:
I don't want to private it only to have to implement most of `VecDeque`'s methods manually. But I do agree regarding `VecDeque`.

Reply:
I do not want to expose the fields, because I don't want field part of the API surface. That is a requirement for this to be merged.

Reply:
What's the minimum API surface you would accept? `push_front`, `push_back`, `remove`?

Reply:
I added those, I'll add more if you request it.